Data Confidentiality in Open Science

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  " Data Confidentiality in Open Science"
+date:   2023-11-06
+author: Konstantinos Rigas
+category: practice
+tags: reusability
+---
+
+Data Confidentiality in Open Science
```

Comment:
This title is not needed as the title is rendered from the metadata.

Reply:
The title has now been removed